src/reducers/index.js

```diff
@@ -1,94 +1,106 @@
-export default (state = {}, action) => {
+import immutable from 'immutable';
+
+const fakeInventory = {
```

Comment:
I'll try and get my Torch server back up this weekend so that you can see the data directly.  I did figure out how to get the server running as a windows service, so I'm 50% of the way there.  Btw, @erindepew do you want a key for the game?  I've got 2 or three of them spare :-)